src/__tests__/__snapshots__/node.test.ts.snap

```diff
@@ -389,12 +383,9 @@ Object {
       "auto": "auto",
       "checkbox": "24px",
       "clearButton": "32px",
-      "logoDefault": "170px",
```

Comment:
I'm not 100% sure logo styles should be extracted. This way if you want to use the logo at the right size you need to include the header package which is not the most intuitive.
Is there a chance we can provide logo component here?

Reply:
in case we need to use the logo somewhere else, we can extract them or use other values, these values are for the header